skiplang/prelude/src/stdlib/collections/persistent/SortedMap.sk

```diff
@@ -536,6 +536,27 @@ base class .SortedMap<+K: Orderable, +V> uses  //  extends KeyedSequence<K, V>
     ValuesIterator::make(this)
   }
 
+  fun itemsAfter<K2: Orderable>[K: K2](start: ?K2): mutable Iterator<(K2, V)> {
```

Comment:
Should this function return a `Map.MapItemsIterator` instead?

Reply:
No. `Map` are mutable hash tables, `SortedMap` are persistent balanced binary trees